frontend/docs/index.mdx

```diff
@@ -0,0 +1,268 @@
+---
+name: Getting started
+route: /
+order: 1
+---
+
+# Trase
+
+[Trase](http://trase.earth) brings unprecedented transparency to commodity supply chains revealing new pathways towards achieving a deforestation-free economy.
+
+<img src="public/trase-screenshot.png" alt="Trase image" width="800"/>
+
+## Project structure
+
+This repository contains the source code for the website in two technically independent applications:
+- `/frontend` contains the frontend application, including a dedicated nodejs server (for dev purposes only);
+- `/` contains the Rails 5.x API that serves data to the above mentioned frontend application.
+
+While technically independent, the frontend application is heavily dependent on the API spec served by the rails application.
+
+## Requirements
+
+### API
+- Ruby 2.4.2
+- Rails 5.1+
+- PostgreSQL 10.6+ with `intarray`, `tablefunc` and `postgres_fdw` extensions
+- [Bundler](http://bundler.io/)
+- redis >= 3
+- wget
+- cron
+
+### Frontend
+- Nodejs 8.2+
+
+## Local setup
+
+### API
+
+We recommend managing your Ruby installation through [rvm](https://github.com/rvm/rvm). Ensure `bundler` gem is installed.
+
+1. Install ruby libraries
+
+    `bundle install`
+
+2. Configure the environment
+
+    Copy `.env.sample` to `.env` and replace the values accordingly. See the Configuration section below for more information.
+
+3. Create the database
+
+    Create an empty database in PostgreSQL and then [import a stable database version](doc/trase/data_export_import.md). If it's an earlier version than the current schema, run the migrations:
```

Comment:
links to the doc folder no longer work; this link should become `docs-backend-data-export-import`